system/src/ble_control_request_channel.h

```diff
@@ -44,7 +44,7 @@
 #define BLE_CHANNEL_DEBUG_ENABLED 0
 #endif
 
-static_assert(BLE_MAX_PERIPH_CONN_COUNT == 1, "Concurrent peripheral connections are not supported");
+//static_assert(BLE_MAX_PERIPH_CONN_COUNT == 1, "Concurrent peripheral connections are not supported");
```

Comment:
`BLE_MAX_PERIPH_CONN_COUNT` -> `BLE_MAX_PERIPHERAL_COUNT` ?

Reply:
We can just remove this assertion, since there is similar assertion in BLE HAL.